packages/plugin-commands-init/test/utils.test.ts

```diff
@@ -0,0 +1,20 @@
+import { workWithInitModule, personToString } from '@pnpm/plugin-commands-init/lib/utils'
+
+test('run the workWithInitModule function', () => {
+  const rawConfig = {
+    initVersion: '2.0.0',
+    initModule: '~/.pnpm-init.js',
```

Comment:
how do you verify that this module was executed?

Reply:
I have made a supplement, please review it